test/common/common.go

```diff
@@ -29,19 +29,19 @@ import (
 )
 
 const (
-	// DockerNginx1230Image is the docker.io nginx:1.23.0 image.
-	DockerNginx1230Image = "docker.io/library/nginx:1.23.0"
-	// DockerNginx1240Image is the docker.io nginx:1.24.0 image.
-	DockerNginx1240Image = "docker.io/library/nginx:1.24.0"
-	// DockerNginx1250Image is the docker.io nginx:1.25.0 image.
-	DockerNginx1250Image = "docker.io/library/nginx:1.25.0"
+	// PublicEcrAwsNginx1230Image is the public.ecr.aws/nginx/nginx:1.23.0 image.
+	PublicEcrAwsNginx1230Image = "public.ecr.aws/nginx/nginx:1.23.0"
+	// PublicEcrAwsNginx1240Image is the public.ecr.aws/nginx/nginx:1.24.0 image.
+	PublicEcrAwsNginx1240Image = "public.ecr.aws/nginx/nginx:1.24.0"
+	// PublicEcrAwsNginx1250Image is the public.ecr.aws/nginx/nginx:1.25.0 image.
+	PublicEcrAwsNginx1250Image = "public.ecr.aws/nginx/nginx:1.25.0"
 
 	// ArtifactRegistryNginx1176Image is the europe-docker.pkg.dev/gardener-project/releases/3rd/nginx:1.17.6 image (copy of docker.io/library/nginx:1.17.6).
 	ArtifactRegistryNginx1176Image = "europe-docker.pkg.dev/gardener-project/releases/3rd/nginx:1.17.6"
 	// RegistryK8sNginx1154Image is the registry.k8s.io/e2e-test-images/nginx:1.15-4 image.
 	RegistryK8sNginx1154Image = "registry.k8s.io/e2e-test-images/nginx:1.15-4"
-	// PublicEcrAwsNginx1199Image is the public.ecr.aws/nginx/nginx:1.19.9 image.
-	PublicEcrAwsNginx1199Image = "public.ecr.aws/nginx/nginx:1.19.9"
+	// GithubRegistryNginx1240Image is the ghcr.io/linuxserver/nginx:1.24.0 image.
+	GithubRegistryNginx1240Image = "ghcr.io/linuxserver/nginx:1.24.0"
```

Comment:
Nit: I would use a tag that does not already exist among our image. Just to prevent potential usage of 2 images with the same tag in the same test.

```suggestion
	GithubRegistryNginx1261Image = "ghcr.io/linuxserver/nginx:1.26.1"
```